be/src/storage/task/engine_checksum_task.cpp

```diff
@@ -77,18 +77,8 @@ Status EngineChecksumTask::_compute_checksum() {
         return Status::OK();
     }
 
-    std::vector<uint32_t> return_columns;
     auto tablet_schema = tablet->tablet_schema();
-
-    size_t num_columns = tablet_schema->num_columns();
-    for (size_t i = 0; i < num_columns; ++i) {
-        LogicalType type = tablet_schema->column(i).type();
-        if (is_support_checksum_type(type)) {
```

Comment:
```
inline bool is_support_checksum_type(LogicalType type) {
    return type != TYPE_FLOAT && type != TYPE_DOUBLE && type != TYPE_HLL && type != TYPE_OBJECT &&
           type != TYPE_PERCENTILE && type != TYPE_JSON;
}
```

Why reach the `ObjectColumn<T>::xor_checksum`?

Reply:
use struct<a bitmap, b bitmap> type in the test case